charts/gardener-extension-provider-azure/templates/configmap-logging.yaml

```diff
@@ -16,7 +16,7 @@ data:
 
     [FILTER]
         Name                parser
-        Match               kubernetes.*machine-controller-manager*azure-machine-controller-manager*
+        Match               kubernetes.*azure-machine-controller-manager*machine-controller-manager-provider-azure*
```

Comment:
The regex here has the following syntax:
kubernetes.*<pod-name-prefix>*<container-name>*

As I see the pod name is still `machine-controller-manager`, so this change will no work.

Reply:
Also now, when there is a second container in the deployment, you should create a new filter which caches it.
You can check the log format of the new container. If it is the same as the other one, you can use the same Parser.
Otherwise I will help you to create a new one

Reply:
So, @Kristian-ZH what is the right thing to put here?

Reply:
Can you please check the log format of the new container via `kubectl logs machine-controller-manager -c azure-machine-controller-manager` and to compare it with the one from `kubectl logs machine-controller-manager machine-controller-manager-provider-azure`

Reply:
The old container here is `azure-machine-controller-manager` and the new container is `machine-controller-manager machine-controller-manager-provider-azure`. Also I don't think there should be any change in logging format as we have just split the single MCM container into two containers now. @AxiomSamarth you can confirm the same by looking at the logs. 

The concern here is the Match filters that are to be used to get the container logs from both containers.

Reply:
I just checked. I do not see any difference in the logging format.

Reply:
okay, then for now, the Filter an be changed to:
```
    [FILTER]
        Name                      parser
        Match                      kubernetes.*machine-controller-manager*azure*
        Key_Name              log
        Parser                     kubeapiserverParser
        Reserve_Data        True
```
This will catch all containers from the machine-controller-manager pod which contains `azure` in their names.
In the future if new containers are added to the pod, we can split this filter to dedicated filters per container.

Reply:
Thank you @Kristian-ZH

Reply:
Okay thanks for the comments and feedback.